examples/with_winit/src/lib.rs

```diff
@@ -136,397 +136,418 @@ fn run(
-                                if let Some(renderer) = &renderers[render_state.surface.dev_id] {
-                                    if let Some(profile_result) = &renderer
-                                        .profile_result
-                                        .as_ref()
-                                        .or(profile_stored.as_ref())
-                                    {
-                                        // There can be empty results if the required features aren't supported
-                                        if !profile_result.is_empty() {
-                                            let path = std::path::Path::new("trace.json");
-                                            match wgpu_profiler::chrometrace::write_chrometrace(
-                                                path,
-                                                profile_result,
-                                            ) {
-                                                Ok(()) => {
-                                                    println!("Wrote trace to path {path:?}");
+                                Key::Named(NamedKey::ArrowRight) => {
+                                    scene_ix = scene_ix.saturating_add(1)
+                                }
+                                Key::Named(NamedKey::ArrowUp) => complexity += 1,
+                                Key::Named(NamedKey::ArrowDown) => {
+                                    complexity = complexity.saturating_sub(1)
+                                }
+                                Key::Named(NamedKey::Space) => {
+                                    transform = Affine::IDENTITY;
+                                }
+                                Key::Character(char) => {
+                                    let char = char.to_lowercase();
+                                    match char.as_str() {
+                                        "q" | "e" => {
+                                            if let Some(prior_position) = prior_position {
+                                                let is_clockwise = char == "e";
+                                                let angle = if is_clockwise { -0.05 } else { 0.05 };
+                                                transform = Affine::translate(prior_position)
+                                                    * Affine::rotate(angle)
+                                                    * Affine::translate(-prior_position)
+                                                    * transform;
+                                            }
+                                        }
+                                        "s" => {
+                                            stats_shown = !stats_shown;
+                                        }
+                                        "d" => {
+                                            complexity_shown = !complexity_shown;
```

Comment:
This no longer works on my machine, but this is also a pre-existing issue from before this PR